docs/docsource/Installation/configure/configuring_zenko.rst

```diff
@@ -0,0 +1,90 @@
+.. _configuring_zenko:
+
+Configuring Zenko
+=================
+
+Zenko is readily configurable using Helm to pass values set in Helm charts. 
+Helm charts are stored in Zenko/kubernetes/zenko/ and its subdirectories.
+Helm charts are YAML files with configurable values and in a Zenko deployment, 
+reconfiguration, or upgrade, Helm reads in the following order:
+
+   #. Base settings for Zenko microservices (for example, Grafana settings,
+      written to Zenko/kubernetes/zenko/charts/grafana/values.yaml).
+   #. Base settings for Zenko. These settings override the base microservice 
+      settings, and are found in Zenko/kubernetes/zenko/values.yaml.
+   #. Custom settings, which you can write to an options.yaml chart. Settings
+      written to this chart override settings read from the preceding
```

Comment:
Same here regarding the use of `chart` in this case it's just a yaml file that we call 'options.yaml' for the sake of example.